code/datums/uplink/devices and tools.dm

```diff
@@ -341,3 +342,9 @@
 			return
 		BS.module.modules += new /obj/item/weapon/bluespace_harpoon/mounted/blitz(BS.module)
 		return TRUE
+
+/datum/uplink_item/item/tools/mindreader
+	name = "Mindreader"
+	desc = "Place on your victim head to extract memory from his brain, after his mental breakdown."
```

Comment:
Should be: "Place on your victim's head to extract memories from their brain after a mental breakdown."

Reply:
Done